### ISSUE TITLE
CMakeLists.txt files missing in source tarball

### ISSUE THREAD

**Issue by GoogleCodeExporter** [timmerk/libnfc#57]
```
The CMakeLists.txt files are (all) missing from the source tarball. It
would be helpful in building libnfc using CMake from the official source
tar instead of needing a SVN checkout.
```

Original issue reported on code.google.com by `fkooman%tuxed.net@gtempaccount.com` on 15 Jan 2010 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
Oh, and it may make sense to include the Doxygen API docs in the source tarball 
as
well so they don't need to be generated anymore. It's fine for SVN releases, 
but for
stable versions it makes sense to already include them.


```

Original comment by `fkooman%tuxed.net@gtempaccount.com` on 15 Jan 2010 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
Hi François,

CMake files are not included with POSIX tarball due to remaining opened issue 
52, and
I haven't enough time to support Autotools and CMake myself.
And there are no CMake files in Windows archive because: I haven't any Windows 
OS, so
I can't maintain it myself, but Roel do... and Roel build it under Windows with 
Makefile.

Regarding, Doxygen API, its a good advice to share documentation in generated 
format
but not in source archive: source is source ^^
BTW, with the upcomming release 1.3.1 we will provide one archive more: HTML
generated documentation.

```

Original comment by `romu...@libnfc.org` on 18 Jan 2010 at 11:16
- Added labels: **Priority-Low**, **OpSys-All**

**Comment by GoogleCodeExporter** [comment]
```
For the Fedora package I switched to using Autotools for now, which works fine. 
Soon
it will be available in Fedora 12 and Fedora Rawhide.

https://bugzilla.redhat.com/show_bug.cgi?id=518779

I guess I'll keep using Autotools then, maybe it makes sense to drop CMake as I 
seem
to be the only one interested. If Windows builds work with Autotools as well, 
there
doesn't seem to be a point to keep it...
```

Original comment by `fkooman%tuxed.net@gtempaccount.com` on 18 Jan 2010 at 12:40

**Comment by GoogleCodeExporter** [comment]
```
CMake files have been experimentally included in 1.3.2, if a maintainer can be 
found
for CMake at least under Windows, we will keep it.
BTW, we really lack of Windows contributors...
```

Original comment by `romu...@libnfc.org` on 1 Feb 2010 at 4:38
- Changed state: **Fixed**
